resources/tools/mdsd/devops/pipeline/stages/impl/dockerized/gradle/modules/Build/Build.groovy

```diff
@@ -0,0 +1,8 @@
+if (CFG.mavenSettingsId) {
```

Comment:
This should be `CFG.gradlePropertiesId`, right?

Reply:
Yes. It worked by chance, as the mavenSettingsId is always set for the MDSDToolsPipeline.